### DISCUSSION TITLE
Confused 🗑️

### DISCUSSION THREAD

**Discussion by peseaj** [anyproto/contributors#251] [Q&A] [answered]
Please, since the last update from the time of this message, when I am in the Graph view and I move an object to bin, it reappears in the Graph. And where did the Bin go to? I can't find it.

**Comment by filip-anytype** [comment, marked as answer]
> Please, since the last update from the time of this message, when I am in the Graph view and I move an object to bin, it reappears in the Graph.

I couldn't replicate this. Please write a bug report on our forum (ideally with a video of it) if you are still able to reproduce this issue.



> And where did the Bin go to?

You'll need to click on the three dots menu in the right corner of the new All Objects system widget.

![image](https://github.com/user-attachments/assets/a10062a1-4ea6-4d58-8179-3ac08a187a14)
